src/main/webapp/app/core/layouts/footer/footer.scss

```diff
@@ -24,4 +17,11 @@
         margin-right: 5px;
         margin-bottom: 5px;
     }
+    .footer-git-responsive {
```

Comment:
Can we addredd this warning? `Expected empty line before rule (rule-empty-line-before)`